experiments/interpretation/dutchf3_patch/local/test.py

```diff
@@ -99,6 +106,21 @@ def reset(self):
         self.confusion_matrix = np.zeros((self.n_classes, self.n_classes))
 
 
+def normalize(array):
+    """
+    Normalizes a segmentation mask array to be in [0,1] range
+    """
+    min = array.min()
+    return (array - min) / (array.max() - min)
+
+
+def mask_to_disk(mask, fname):
+    """
+    write segmentation mask to disk using a particular colormap
+    """
+    Image.fromarray(cm.gist_earth(normalize(mask), bytes=True)).save(fname)
+
+
 def _transform_CHW_to_HWC(numpy_array):
     return np.moveaxis(numpy_array, 0, -1)
```

Comment:
I think these should be in a `utils` file somewhere, i've seen `_transform_CHW_to_HWC()` copied in mutliple places. Doesn't have to go in this PR, but something we should keep in mind.